subprojects/packagefiles/svtjpegxs/meson.build

```diff
@@ -0,0 +1,33 @@
+project(
+  'svtjpegxs',
+  'c',
+  version: '0.9.0',
+  meson_version: '>= 0.64.0',
+)
+
+cc = meson.get_compiler('c')
+if host_machine.cpu_family() == 'x86_64'
+  add_project_arguments(
+    '-DARCH_X86_64=1',
+    language: 'c',
+  )
+else
+  error('unsupported architecture: SVT-JPEG-XS requires x86_64')
```

Comment:
Bit confused by this. Are you saying you're only supporting x86_64 for now because that's the only thing you care about?

It should build fine on other architectures, with the C implementation.

In GStreamer cerbero we build it for all architectures I think, and we only enable the assembly for x86 + x86_64.

Reply:
The upstream readme may or may not be accurate but it does say x86_64 is required.

Reply:
> In GStreamer cerbero we build it for all architectures I think, and we only enable the assembly for x86 + x86_64.

That might not be quite right actually. I was only looking at the svt-jpeg-xs recipe, but it's only added as a dep in the  gst-plugins-bad recipe in case of x86_64.

Reply:
> It should build fine on other architectures, with the C implementation.

I don't think they have C implementation, from what I see it is x86_64 only.